### PR TITLE
Hotfix Finish support without parameter hotfixBranch

### PR DESCRIPTION
**Pull request by basmussen**
Implementation of ticket #326. This allows to close a hotfix branch in batch mode without specifying the parameter hotfixBranch, if only one hotfix branch is available.

### CODE REVIEW

File: src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java [hunk 1]
```diff
@@ -154,6 +154,31 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     throw new MojoFailureException("Hotfix branch with name '" + branch + "' doesn't exist. Cannot finish hotfix.");
                 }
                 hotfixBranchName = branch;
+            } else if (StringUtils.isBlank(hotfixVersion)) {
```
Comment (beatngu13) [inline src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java:157]: **DISCLAIMER:** I'm not a maintainer, so feel free to ignore what I'm saying. 😉

This is basically the same block as in [`GitFlowReleaseFinishMojo`](https://github.com/aleksandr-m/gitflow-maven-plugin/blob/master/src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowReleaseFinishMojo.java#L191-L218).

I wonder if it makes sense / is possible to avoid the duplicated code by, e.g., having an intermediate layer like `AbstractGitFlowFinishMojo`, which encloses this functionality. (Although it looks like `GitFlowFeatureFinishMojo` couldn't benefit from it.)

Comment (levitin) [reply src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java:157]: Abstraction via inheritance is an obvious improvement since you write less code, right? But in many cases inheritance becomes much more inconvenient to maintain, than just copy some code, that has potential to diverge a little in the future.

If any, I would prefer using composition over inheritance everywhere. At least you can interchange the solution with a very less effort.

**Comment by levitin** [comment]
@aleksandr-m: Any news on this issue? The solution is already there? Can you review and merge it?

File: src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java (hunk 1 above)
Comment (beatngu13) [reply src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java:157]: >Abstraction via inheritance is an obvious improvement since you write less code, right?

No, but thanks for ELI5. 😉

**Comment by basmussen** [comment]
I can confirm that both mvn gitflow:hotfix-start and mvn gitflow:hotfix-finish work successfully with version 1.21 using the following configuration:

```
  mvn gitflow:hotfix-start -B -ntp -DhotfixVersion=1.21 -DargLine="-B -ntp"
  mvn gitflow:hotfix-finish -B -ntp -DhotfixVersion=1.21 -DargLine="-B -ntp"
```

  This batch mode execution with the specified parameters functions as expected. Great addition - this will definitely streamline our CI/CD workflows!

  With this enhancement, this PR is no longer necessary as the functionality now works in batch mode.
